src/components/Data/Documents/ViewMap.vue

```diff
@@ -39,7 +39,7 @@
             <li>
               <a
                 title="You are not allowed to delete this document"
-                :class="{ 'text-secondary': !canDelete }"
+                :class="{ disabled: !canDelete }"
```

Comment:
This is a little mistake I made while resolving conflicts with the `4-dev` branch. But this component is currently deactivated. I propose that we leave this as-is and we'll focus on the component once we restore it.